src/ui.rs

```diff
@@ -2138,7 +2175,7 @@ where
                     );
                     paint_default_icon(ui, openness, &r);
 
-                    if r.clicked_by(PointerButton::Primary) {
+                    if r.clicked_by(config.click_header.mouse_button) {
```

Comment:
Is there option in egui that dictates which button clocks buttons? Because that's a button.
If buttons are always clicked by primary - no need to configure it here. It would be confusing for the user.